client/src/lib/GlobalStyles.js

```diff
@@ -165,6 +165,13 @@ const GlobalStyle = createGlobalStyle`
     font-size: 1.1rem;
```

Comment:
Let's get rid of these legacy styles - the `color`, `border`, `font-size`, `margin`. `border-radius` can be increased to 4px.